install/helm/agones/values.yaml

```diff
@@ -216,6 +216,8 @@ agones:
     controller:
       name: agones-controller
       pullPolicy: IfNotPresent
+    extensions:
```

Comment:
Still add a comment, and indent to the same level as `extensions`, e.g.:

```
    # extensions settings ignored unless `SplitControllerAndExtensions` feature gate is enabled
    extensions:
```

Reply:
Done.

Reply:
(Done)